src/library/scala/collection/mutable/ArrayDeque.scala

```diff
@@ -51,22 +51,22 @@ class ArrayDeque[A] protected (
 
   def this(initialSize: Int = ArrayDeque.DefaultInitialSize) = this(ArrayDeque.alloc(initialSize), start = 0, end = 0)
 
-  def apply(idx: Int) = {
+  override def apply(idx: Int) = {
```

Comment:
we ususally don't write `override` for implementations of abstract methods

Reply:
Isn't putting `override` a good idea? If the parent removes the abstract method, it would be a compile error instead of being left with an orphan method?

Reply:
It's a matter of taste, I personally see it like https://stackoverflow.com/a/5643284/248998. In this repo we use `override` when actually overriding.

Reply:
This convention put a lot of burden on the author of the parent - should she decide to provide a default implementation one day, she would have to hunt down all the subclasses and remove the override keyword. But, as you said, its a matter of taste - I don't have a strong opinion here since my contribution to this repository is tiny :)